web/auth/oauth.go

```diff
@@ -238,9 +239,57 @@ func (a *AuthorizeHTTPHandler) authorizeForm(c echo.Context) error {
 		return c.Redirect(http.StatusFound, u.String()+"#")
 	}
 
+	if !params.client.Flagship {
+		flags, err := feature.GetFlags(inst)
+		if err != nil {
+			return err
+		}
+
+		if clientsLimit, ok := flags.M["cozy.oauthclients.max"].(float64); ok && clientsLimit >= 0 {
+			limit := int(clientsLimit)
+
+			clients, _, err := oauth.GetConnectedUserClients(inst, 100, "")
```

Comment:
If we use `limit + 1` and we have say, 5 OAuth clients for a limit of 3, the modal will display `4/3 clients connected` instead of `5/3 clients connected`.